provider/rfc2136/rfc2136.go

```diff
@@ -92,11 +109,11 @@ var tsigAlgs = map[string]string{
 
 type rfc2136Actions interface {
 	SendMessage(msg *dns.Msg) error
-	IncomeTransfer(m *dns.Msg, a string) (env chan *dns.Envelope, err error)
```

Comment:
a is not really specific on what it is or what it does, this can be changed back if needed